### PR TITLE
Debezium UI

### PR DESCRIPTION
**Pull request by indraraj**

### CODE REVIEW

File: example/demo/docker-compose.yaml [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+      - MYSQL_PASSWORD=mysql
+    networks:
+      - dbzui-network
+  dbzui-db-mongo:
+    container_name: dbzui-db-mongo
+    image: mongo:3.6
+    command: ["mongod", "--replSet", "rs0", "--auth"]
+    ports:
+      - "37017:27017"
+    volumes:
```
Comment (indraraj) [inline example/demo/docker-compose.yaml:46]: Hey @rk3rn3r  I am not sure about this.

Comment (rk3rn3r) [reply example/demo/docker-compose.yaml:46]: should be fine

File: example/demo/README.md [hunk 2]
```diff
@@ -0,0 +1,82 @@
+# Debezium UI Example
+
+This demo automatically deploys the topology of services as defined in the [Debezium Tutorial](https://debezium.io/docs/tutorial/) along with Debezium UI.
```
Comment (gunnarmorling) [inline example/demo/README.md:3]: Let's link to the right URL, not go through the redirect.

File: example/demo/README.md [hunk 3]
```diff
@@ -0,0 +1,82 @@
...
+
+
+## Prerequisites
+
+Debezium UI is only available with Debezium version 1.5 and above.
+
+
+## Launching UI
+
+Launch all the required component
```
Comment (gunnarmorling) [inline example/demo/README.md:13]:
```suggestion
Launch all the required components:
```

File: example/demo/README.md [hunk 4]
```diff
@@ -0,0 +1,82 @@
...
+Creating dbzui-connect   ... done
+
+```
+
+
+Debezium UI will be available on http://localhost:8080
+
+## Register connectors
+
+Registration of the Postgres, Mongo DB, MySQL & SQL server(coming soon) connectors can be done using [Debezium UI](http://localhost:8080). The first two steps of *Create connector* wizard in the Debezium UI are mandatory i.r `` Connector type `` & `` Properties step`` (where all the required basic connection properties are entered).
```
Comment (gunnarmorling) [inline example/demo/README.md:35]:
```suggestion
Registration of the Postgres, Mongo DB, MySQL & SQL Server (coming soon) connectors can be done using [Debezium UI](http://localhost:8080). The first two steps of *Create connector* wizard in the Debezium UI are mandatory (i.e. the "Connector type" and "Properties" steps, where all the required basic connection properties are entered).
```

File: example/demo/README.md [hunk 5]
```diff
@@ -0,0 +1,82 @@
...
+
+Debezium UI will be available on http://localhost:8080
+
+## Register connectors
+
+Registration of the Postgres, Mongo DB, MySQL & SQL server(coming soon) connectors can be done using [Debezium UI](http://localhost:8080). The first two steps of *Create connector* wizard in the Debezium UI are mandatory i.r `` Connector type `` & `` Properties step`` (where all the required basic connection properties are entered).  
+
+### MySQL
+
+Required values **Hostname**: dbzui-db-mysql, **User**: debezium, **Password**: dbz, **Kafka broker addresses**: kafka:9092, **Database history topic name**: dbhistory.inventory.
```
Comment (gunnarmorling) [inline example/demo/README.md:39]: Perhaps add a screenshot?

File: example/demo/README.md [hunk 6]
```diff
@@ -0,0 +1,82 @@
...
+
+
+To connect with MySQL Cli clinet
```
Comment (gunnarmorling) [inline example/demo/README.md:42]:
```suggestion
To connect with MySQL Cli client:
```

Comment (gunnarmorling) [reply example/demo/README.md:42]: Same typo below twice.

File: example/demo/README.md [hunk 7]
```diff
@@ -0,0 +1,82 @@
...
+mongo -u debezium -p dbz --authenticationDatabase admin localhost:37017/inventory
+
+```
+
+
+## Examine the change events
+
+```shell
+# Open in a new terminal
+# Viewing the change events in kafka dbzui-kafka containner
```
Comment (gunnarmorling) [inline example/demo/README.md:75]:
```suggestion
# Viewing the change events in kafka dbzui-kafka container
```

We use kafkacat preferably, via the debezium-tooling image. See other recent examples, e.g. the ones for _saga_ and _fk-join_.

File: example/demo/docker-compose.yaml [hunk 8]
```diff
@@ -0,0 +1,90 @@
...
+      - dbzui-db-pg
+    environment:
+      - BOOTSTRAP_SERVERS=dbzui-kafka:9092
+      - GROUP_ID=1
+      - CONFIG_STORAGE_TOPIC=my_connect_configs
+      - OFFSET_STORAGE_TOPIC=my_connect_offsets
+      - STATUS_STORAGE_TOPIC=my_connect_statuses
+    networks:
+      - dbzui-network
+  dbzui-debezium-ui:
```
Comment (gunnarmorling) [inline example/demo/docker-compose.yaml:77]: Do we need the "dbzui-" prefix in all the service names? Seems it could simply be omitted?

**Review by gunnarmorling — COMMENTED** [review]
Nice one! A few comments inline.

File: example/demo/docker-compose.yaml (hunk 8 above)
Comment (indraraj) [reply example/demo/docker-compose.yaml:77]: yeah, I think it can be omitted

File: example/demo/README.md (hunk 5 above)
Comment (indraraj) [reply example/demo/README.md:39]: ok

File: example/demo/README.md (hunk 2 above)
Comment (indraraj) [reply example/demo/README.md:3]: sure

**Comment by mdrillin** [comment]
I think this would be better under a folder like 'ui-demo' instead of 'examples/demo'.  Also, I'm having problems getting this come up - will contact you on chat to work thru issues.

File: example/demo/docker-compose.yaml [hunk 9]
```diff
@@ -0,0 +1,90 @@
+version: '1'
```
Comment (mdrillin) [inline example/demo/docker-compose.yaml:1]: I think this should be version: '3' to agree with other yamls

**Comment by mdrillin** [comment]
@indraraj It seemed easier to open a different PR for this one - I incorporated requested changes plus some reorganization. #158

**Comment by indraraj** [comment]
Continuing in different PR https://github.com/debezium/debezium-examples/pull/158 , closing this one.

**Pull request closed without merging**